frontend/appflowy_flutter/packages/flowy_svg/bin/flowy_svg.dart

```diff
@@ -0,0 +1,272 @@
+import 'dart:async';
+import 'dart:developer';
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:path/path.dart' as path;
+
+import 'options.dart';
+
+const languageKeywords = [
+  'abstract',
+  'else',
+  'import',
+  'show',
+  'as',
+  'enum',
+  'static',
+  'assert',
+  'export',
+  'interface',
+  'super',
+  'async',
+  'extends',
+  'is',
+  'switch',
+  'await',
+  'extension',
+  'late',
+  'sync',
+  'base',
+  'external',
+  'library',
+  'this',
+  'break',
+  'factory',
+  'mixin',
+  'throw',
+  'case',
+  'false',
+  'new',
+  'true',
+  'catch',
+  'final',
+  'variable',
+  'null',
+  'try',
+  'class',
+  'final',
+  'class',
+  'on',
+  'typedef',
+  'const',
+  'finally',
+  'operator',
+  'var',
+  'continue',
+  'for',
+  'part',
+  'void',
+  'covariant',
+  'Function',
+  'required',
+  'when',
+  'default',
+  'get',
+  'rethrow',
+  'while',
+  'deferred',
+  'hide',
+  'return',
+  'with',
+  'do',
+  'if',
+  'sealed',
+  'yield',
+  'dynamic',
+  'implements',
+  'set',
+];
+void main(List<String> args) {
+  if (_isHelpCommand(args)) {
+    _printHelperDisplay();
+  } else {
+    generateSvgData(_generateOption(args));
+  }
+}
+
+bool _isHelpCommand(List<String> args) {
+  return args.length == 1 && (args[0] == '--help' || args[0] == '-h');
+}
+
+void _printHelperDisplay() {
+  final parser = _generateArgParser(null);
+  log(parser.usage);
+}
+
+Options _generateOption(List<String> args) {
+  final generateOptions = Options();
+  _generateArgParser(generateOptions).parse(args);
+  return generateOptions;
+}
+
+ArgParser _generateArgParser(Options? generateOptions) {
+  final parser = ArgParser()
+    ..addOption(
+      'source-dir',
+      abbr: 'S',
+      defaultsTo: '/assets/images',
+      callback: (String? x) => generateOptions!.sourceDir = x,
+      help: 'Folder containing localization files',
+    )
+    ..addOption(
+      'output-dir',
+      abbr: 'O',
+      defaultsTo: '/lib/generated',
+      callback: (String? x) => generateOptions!.outputDir = x,
+      help: 'Output folder stores for the generated file',
+    )
+    ..addOption(
+      'name',
+      abbr: 'N',
+      defaultsTo: 'flowy_svgs.g.dart',
+      callback: (String? x) => generateOptions!.outputFile = x,
+      help: 'The name of the output file that this tool will generate',
+    );
```

Comment:
these options are configured so that `dart run flowy_svg` should work out of the box